### PR TITLE
chore: remove locale subpath allowed urls

### PR DESCRIPTION
**Pull request by danielgent**
Now we're using the `lang` querystring param everywhere, I think we're good to go with this :sunglasses:

### CODE REVIEW

File: src/utils/validate-redirect-url.ts [hunk 1]
```diff
@@ -39,85 +39,28 @@ function matchHostnameWithWildcards(
...
-  "https://*.vercel.sesamy.dev/nb",
-  "https://*.vercel.sesamy.dev/it",
   "https://*.vercel.sesamy.dev/callback",
-  "https://*.vercel.sesamy.dev/sv/callback",
-  "https://*.vercel.sesamy.dev/nb/callback",
-  "https://*.vercel.sesamy.dev/it/callback",
-  "https://*.vercel.sesamy.dev/sv/enter-code",
-  "https://*.vercel.sesamy.dev/nb/enter-code",
-  "https://*.vercel.sesamy.dev/it/enter-code",
+  "https://*.vercel.sesamy.dev/enter-code",
```
Comment (danielgent) [inline src/utils/validate-redirect-url.ts:61]: lol. this was missing for preview deploys, we only had the locale ones :smile:

File: src/app.ts [hunk 2]
```diff
@@ -18,9 +18,7 @@ const ALLOWED_ORIGINS = [
   "https://login2.sesamy.dev",
   "https://login2.sesamy.dev",
   "https://*.vercel.sesamy.dev",
-  "https://auth-admin.sesamy.dev",
   "https://login2.sesamy.com",
-  "https://auth-admin.sesamy.com",
```
Comment (danielgent) [inline src/app.ts:23]: auth-admin only goes through the token service... right? :thinking: 

*AH* or is this for API calls?

**Review by markusahlstrand — APPROVED** [review]
Looks good!

**Comment by danielgent** [comment]
Is this safe @markusahlstrand ? :smile: 

It's where we want to go anyway! :sunglasses:

**Pull request merged**